.eslintrc

```diff
@@ -0,0 +1,230 @@
+{
+  "plugins": [
+    "eslint-plugin-html"
+  ],
+
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true,
+    "atomtest": true,
+    "amd": true,
+  },
+
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "arrowFunctions": true,
+      "binaryLiterals": true,
+      "blockBindings": true,
+      "classes": true,
+      "defaultParams": true,
+      "destructuring": true,
+      "forOf": true,
+      "generators": true,
+      "modules": true,
+      "objectLiteralComputedProperties": true,
+      "objectLiteralDuplicateProperties": true,
+      "objectLiteralShorthandMethods": true,
+      "objectLiteralShorthandProperties": true,
+      "octalLiterals": true,
+      "regexUFlag": true,
+      "regexYFlag": true,
+      "spread": true,
+      "superInFunctions": true,
+      "templateStrings": true,
+      "unicodeCodePointEscapes": true,
+      "globalReturn": true,
+      "jsx": true
+    }
+  },
+
+  "globals": {
+    "atom": true
+  },
+
+  // http://eslint.org/docs/rules/
+  "rules": {
```

Comment:
I have not filtered through all this rules. I modified most of the ones that showed errors, but we may need to edit this a bit in the future to have better style.